basket/news/backends/acoustic.py

```diff
@@ -7,8 +7,6 @@
 from requests import ConnectionError
 from silverpop.api import Silverpop, SilverpopResponseException
 
-from basket import metrics
```

Comment:
Where was this used? I don't see it in removed code so do we still need it?

Reply:
It was used down below on L127. The other code in this file don't use it so it can be removed.

Reply:
d'oh serves me right for reviewing on GH Mobile